.github/ISSUE_TEMPLATE/1.bug.yml

```diff
@@ -8,20 +8,22 @@ body:
         Please provide a searchable summary of the issue in the title above ⬆️.
 
         Thanks for contributing by creating an issue! ❤️
-  - type: checkboxes
+
+  - type: input
     attributes:
-      label: Duplicates
-      description: Please [search the history](https://github.com/mui/mui-x/issues) to see if an issue already exists for the same problem.
-      options:
-        - label: I have searched the existing issues
-          required: true
+      label: Search keywords
+      description: Your issue may already have been reported. Please list the keywords you've used to search the [existing issues](https://github.com/mui/mui-x/issues). This will allow us to improve the experience for other users.
+      placeholder: ex. datagrid column resizing
+    validations:
```

Comment:
It feels like most people will enter "sefsef" in this field, me the first, even if I'm searching first for GitHub issues that are duplicates. It's painful to have to type it again here, you might not remember what you typed. And for the people who would skip it, I doubt they will search. But experimenting with it sounds interesting, we will learn something either way

Reply:
Yeah I figured it costs nothing to try it for a few weeks. What would really be useful is if github could provide a list of suggested duplicates as the user types their issue title.